runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingPlugin.java

```diff
@@ -133,7 +133,7 @@ public void onChatMessage(ChatMessage event)
 	{
 		if (event.getType() == ChatMessageType.SPAM || event.getType() == ChatMessageType.GAMEMESSAGE)
 		{
-			if (event.getMessage().startsWith("You get some") && (event.getMessage().endsWith("logs.") || event.getMessage().endsWith("mushrooms.")))
+			if (event.getMessage().matches("You get (?:some|an)[\\w ]+(?:logs?|mushrooms)\\."))
```

Comment:
precompile the pattern and save it as constant at top of the file

Reply:
Good idea.